### DISCUSSION TITLE
[NHẬN TASK]

### DISCUSSION THREAD

**Discussion by nguyenxuanquang6789** [Runway-Club/Runway-CV-Builder#24] [Task Delivery]
Cho mình lấy task: Tạo giao diện dialog Skill, Education, Hobby, Employments của Trang chỉnh sửa CV

**Comment by nguyentantrung2000** [comment]
Sau khi xem xét thì task này vẫn chưa có người nhận, nên bạn sẽ làm task này nhé :
Thông tin của task:
1. Dùng để người dùng chọn các thông tin trên CV.
2. Làm bằng các dialog nhỏ.
Yêu cầu của task:
1. Thiết kế phù hợp đúng bố cục figma
2. Có thể ấn chọn và thêm các options
3. .deadline 11:59 PM 30/7/2021
Nếu bạn đồng ý vui lòng reply cho mình biết nhé

**Reply by nguyenxuanquang6789** [reply]
Mình đồng ý
